docs/user/plagiarism-check.rst

```diff
@@ -198,6 +198,15 @@ The following activity diagram depicts the CPC flow described above.
 
 |cpc-activity-diagram|
 
+
+This video below shows the full CPC process, including creation of a new exercise with the CPC enabled, viewing checks results and post due date steps:
```

Comment:
Here, you can add roles into this sentence as well as you did for the PR description, like by a student or instructor. It would be better to understand what to watch beforehand.